### PR TITLE
[ZH-CN] Translations

### PR DESCRIPTION
**Pull request by ilyaspiridonov**
Translated by Alconost

### CODE REVIEW

File: site/zh-cn/federated/tutorials/custom_federated_algorithms_1.ipynb [hunk 1]
```diff
@@ -463,15 +458,15 @@
         "\n",
         "在 TFF 代码的主体内，根据设计，无法枚举构成由 `tff.CLIENTS` 表示的组的设备，也无法探测组中是否存在某个特定设备。在 Federated Core API、基础架构抽象集或我们提供的用于支持模拟的核心运行时基础结构中，没有任何设备或客户端标识的概念。您编写的所有计算逻辑都将表达为在整个客户端组上的运算。\n",
         "\n",
-        "回想一下我们前面提到的联合类型的值与 Python `dict` 的不同，因为它不能简单地枚举其成员组成。将您的 TFF 程序逻辑所操作的值视为与布局（组）所关联，而不是与单个参与者相关联。\n",
+        "Recall here what we mentioned earlier about values of federated types being unlike Python `dict`, in that one cannot simply enumerate their member constituents. Think of values that your TFF program logic manipulates as being associated with placements (groups), rather than with individual participants.\n",
```
Comment (joefernandez) [inline site/zh-cn/federated/tutorials/custom_federated_algorithms_1.ipynb:461]: Seems like we are missing a translation here.

Comment (ilyaspiridonov) [reply site/zh-cn/federated/tutorials/custom_federated_algorithms_1.ipynb:461]: @joefernandez I can see it translated in the GitLocalize UI, which suggests that in selected cases some segments somehow lose translations at the reverse parsing stage when we manually download the file. Hm, will talk to the dev team about that.

Comment (ilyaspiridonov) [reply site/zh-cn/federated/tutorials/custom_federated_algorithms_1.ipynb]: @joefernandez this is fixed, thank you 
@pcoet

**Comment by ilyaspiridonov** [comment]
@pcoet

**Pull request merged**